components/centraldashboard/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "kubeflow-centraldashboard",
+  "version": "0.4.0",
+  "description": "Kubeflow Central Dashboard",
+  "scripts": {
+    "build": "NODE_ENV=production npx webpack && npm run tslint && npm run build-ts",
+    "build-ts": "tsc",
+    "build-webpack": "npx webpack",
+    "debug": "npm run build && npm run watch-debug",
+    "dev": "concurrently -k -p \"[{name}]\" -n \"Webpack,TypeScript,Express\" -c \"green.bold,yellow.bold,cyan.bold\" \"npx webpack-dev-server\" \"npm run watch-ts\" \"npm run watch-node\"",
+    "dev-debug": "concurrently -k -p \"[{name}]\" -n \"Webpack,TypeScript,Express\" -c \"green.bold,yellow.bold,cyan.bold\" \"npx webpack-dev-server\" \"npm run watch-ts\" \"npm run serve-debug\"",
+    "serve": "node dist/server.js",
+    "serve-debug": "nodemon --inspect dist/server.js",
```

Comment:
Maybe we should mention `nodemon` as a global dependency?
```
npm i -g nodemon
```

Reply:
It's included in devDependencies.